lumigator/jobs/inference/inference.py

```diff
@@ -139,16 +146,8 @@ def run_inference(config: InferenceJobConfig, api_key: str | None = None) -> Pat
             avg_reasoning_tokens=avg_reasoning_tokens,
             avg_answer_tokens=avg_answer_tokens,
         )
```

Comment:
What if this Pydantic model has some sort of __init__ logic that creates the average metrics or throws an error if there are any issues? idk